docs/integrations/editors.md

```diff
@@ -78,20 +78,46 @@ Options include the following:
 
 ## Wing IDE
 
-Wing supports black via the OS Commands tool, as explained in the Wing documentation on
-[pep8 formatting](https://wingware.com/doc/edit/pep8). The detailed procedure is:
+Wing supports `black` via the **Preference Setting**, **Project Properties**, and **OS
+Commands tool**, as explained in the Wing documentation on
+[Auto-Reformatting](https://wingware.com/doc/edit/auto-reformatting). The detailed
+procedure is:
 
-1. Install `black`.
+### Prequistes
```

Comment:
```suggestion
### Prerequisites
```
Interesting my spell checker didn't catch this :thinking:

Reply:
Fixed :smile: